.github/workflows/build.yml

```diff
@@ -16,19 +16,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.3', '7.4']
+        php-versions: ['8.0']
```

Comment:
This is just until we get the tests to correctly ignore the PHP8 implementations in 7.x

Reply:
```suggestion
        php-versions: ['7.3', '7.4', '8.0']
```